app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+                                           f'burst_runconfig_{burst_id}.yaml')
+
+        runconfig_dict_out = runconfig_dict_in.copy()
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'input_file_group',
+                                 'burst_id'],
+                                [burst_id])
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'processing',
+                                 'geocoding',
+                                 'memory_mode'],
+                                'single_block')
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_dir'],
+                                 output_dir_child)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                    ['runconfig',
+                                    'groups',
+                                    'product_group',
+                                    'scratch_path'],
+                                    os.path.join(cfg_in.groups.product_group.scratch_path,
+                                                 f'{os.path.basename(output_dir_child)}_child_scratch')
+                                    )
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'save_secondary_layers_as_hdf5'],
+                                 False)
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'output_imagery_format'],
+                                 'GTiff')
+
+        set_dict_item_recursive(runconfig_dict_out,
+                                ['runconfig',
+                                 'groups',
+                                 'product_group',
+                                 'save_mosaics'],
+                                False)
+
+        # TODO Think if it would be necessary to always turn on `save_bursts` for the child runconfigs?
+        if cfg_in.groups.product_group.save_mosaics:
+            set_dict_item_recursive(runconfig_dict_out,
+                                    ['runconfig',
+                                     'groups',
+                                     'processing',
+                                     'geocoding',
+                                     'save_nlooks'],
+                                    True)
```

Comment:
Just please add a "TODO" here that this line should be removed once mosaicking doesn't require nlooks anymore